### PR TITLE
fix: for symfony 6 and php8.1

### PR DESCRIPTION
**Pull request by aszenz**
Required before fixing tests for graphlite-bundle
I suggest we fix this pkg first

### CODE REVIEW

**Comment by aszenz** [comment]
Relates to https://github.com/thecodingmachine/graphqlite-bundle/pull/174

File: src/ConstraintViolationException.php [hunk 1]
```diff
@@ -28,24 +28,14 @@ public function isClientSafe(): bool
         return true;
     }
 
-    /**
-     * Returns string describing a category of the error.
-     *
-     * Value "graphql" is reserved for errors produced by query parsing or validation, do not use it.
-     */
-    public function getCategory(): string
```
Comment (aszenz) [inline src/ConstraintViolationException.php:36]: `ClientAware` no longer has this method now, see https://github.com/webonyx/graphql-php/pull/830 for why they removed the category

File: src/ConstraintViolationException.php [hunk 2]
```diff
@@ -28,24 +28,14 @@ public function isClientSafe(): bool
...
-
     /**
      * Returns the "extensions" object attached to the GraphQL error.
      *
      * @return array<string, mixed>
      */
     public function getExtensions(): array
     {
-        $extensions = [];
+        $extensions = ['category' => 'Validate'];
```
Comment (aszenz) [inline src/ConstraintViolationException.php:38]: To keep tests running and for backwards compatibility i added category to extensions manually

**Comment by nguyenk** [comment]
Thank you so much @aszenz !!

**Pull request merged**